Dockerfile

```diff
@@ -1,15 +1,19 @@
-FROM golang:1.11-alpine AS builder
+FROM golang:1.13-alpine AS builder
+
+WORKDIR /project
```

Comment:
what do we get from changing this?

Reply:
Just working outside the `$GOPATH` to "proofe" that the switch to go modules work